webroot/adm/keysets.html

```diff
@@ -118,7 +118,12 @@ <h3>Output</h3>
 
         let payload = {keyset_id: keysetId}
 
-        if((add && add.length !== 0) || (remove && remove.length !== 0)) {
+        if ($('#nullify').is(':checked')){
+          allowedSites = null
+        } else if((add && add.length !== 0) || (remove && remove.length !== 0)) {
+          if(allowedSites !== undefined && allowedSites == null) {
+            $('#errorOutput').html('cannot set siteIds for null allowed_sites keyset. Please reset to [] first')
```

Comment:
- I find the "Reset Keyset" terminology quite cryptic. Can we change it to something like "Reset allowed_sites to empty"?
- Also the UX for resetting to empty is quite different from resetting to null. Do we want to make them more similar?

Reply:
I've changed this to a ticket UID2-1598 so that we can close this PR. 
This PR looks good - please merge!